### ISSUE TITLE
Can not turn off early_stop flag

### ISSUE THREAD

**Issue by terrylau2013** [mozilla/DeepSpeech#1540]
For support and discussions, please use our [Discourse forums](https://discourse.mozilla.org/c/deep-speech).

If you've found a bug, or have a feature request, then please create an issue with the following information:

- **Have I written custom code (as opposed to running examples on an unmodified clone of the repository)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: MacOS
- **TensorFlow installed from (our builds, or upstream TensorFlow)**: Official
- **TensorFlow version (use command below)**: v1.6.0-0-gd2e24b6039 1.6.0
- **Python version**: 3.6.6
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:
- **DeepSpeech version**: 0.2.0-alpha.9

You can obtain the TensorFlow version with

I changed the early_stop flag to Flase and set run para early_stop to Flase, but I still see FLAGS.early_stop is True.

`tf.app.flags.DEFINE_boolean ('early_stop',       False,        'enable early stopping mechanism over validation dataset. Make sure that dev FLAG is enabled for this to work')
`
(deepspeech-venv) Laus-MacBook-Pro:DeepSpeech-master lauluenwing$ python -u DeepSpeech.py \
>   --train_files /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/data/chinese/chinese.csv \
>   --dev_files /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/data/chinese/dev.csv \
>   --test_files /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/data/chinese/test.csv \
>   --train_batch_size 1 \
>   --dev_batch_size 1 \
>   --test_batch_size 1 \
>   --n_hidden 2048 \
>   --epoch 75 \
>   --validation_step 1 \
>   --dropout_rate 0.005 \
>   --learning_rate 0.0001 \
>   --early_stop False \
>   --report_count 100 \
>   --use_seq_length False \
>   --export_dir /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/results/model_export/ \
>   --checkpoint_dir /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/results/checkout/ \
>   --decoder_library_path /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/native_client/libctc_decoder_with_kenlm.so \
>   --alphabet_config_path /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/data/chinese/alphabet.txt \
>   --lm_binary_path /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/data/chinese/lm.binary \
>   --lm_trie_path /Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/data/chinese/trie \
>   "$@"
/Users/lauluenwing/tmp/deepspeech-venv/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
/Users/lauluenwing/tmp/deepspeech-venv/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
/Users/lauluenwing/Documents/workspace/DeepSpeech/DeepSpeech-master/util/audio.py:17: DeepSpeechDeprecationWarning: DeepSpeech Python bindings could not be imported, resorting to slower code to compute audio features. Refer to README.md for instructions on how to install (or build) the DeepSpeech Python bindings.
  category=DeepSpeechDeprecationWarning)
I STARTING Optimization
I Training epoch 0...
100% (1 of 1) |###########################################################################################################################| Elapsed Time: 0:00:00 ETA:  00:00:00I Training of Epoch 0 True - loss: 603.018066
100% (1 of 1) |###########################################################################################################################| Elapsed Time: 0:00:01 Time:  0:00:01
I Validating epoch 0...
100% (1 of 1) |###########################################################################################################################| Elapsed Time: 0:00:00 ETA:  00:00:00I Validation of Epoch 0 - loss: 112.888100
100% (1 of 1) |###########################################################################################################################| Elapsed Time: 0:00:32 Time:  0:00:32
I Training epoch 1...
100% (1 of 1) |###########################################################################################################################| Elapsed Time: 0:00:00 ETA:  00:00:00I Training of Epoch 1 True - loss: 121.105156
100% (1 of 1) |###########################################################################################################################| Elapsed Time: 0:00:01 Time:  0:00:01
I Validating epoch 1...
100% (1 of 1) |###########################################################################################################################| Elapsed Time: 0:00:00 ETA:  00:00:00I Validation of Epoch 1 - loss: 123.591736
100% (1 of 1) |###########################################################################################################################| Elapsed Time: 0:00:32 Time:  0:00:32

**Comment by kdavis-mozilla** [comment]
Try `--noearly_stop`

**Comment by terrylau2013** [comment]
@kdavis-mozilla Thanks! this work! Is it TensorFlow standard? All boolean flag use 'no' to set to false?

**Comment by kdavis-mozilla** [comment]
Welcome, and yes it's a TensorFlow "thing" as we're using their command line parser[[1](https://github.com/mozilla/DeepSpeech/blob/master/DeepSpeech.py#L143)].
